cmd/server/main.go

```diff
@@ -83,26 +81,14 @@ func runService(c *cli.Context) error {
 	killChan := make(chan os.Signal, 1)
 	signal.Notify(killChan, syscall.SIGINT)
 
-	svc := service.NewService(conf, psrpcClient, bus)
-
-	sipSrv := sip.NewServer(conf, svc.HandleTrunkAuthentication, svc.HandleDispatchRules)
```

Comment:
@dennwc the split of ‘service’ and ‘sip service’ comes from ingress/egress. I think it’s best to keep it that way so things feel similar

Reply:
Updated. Is this what you had in mind?